source/client-side-encryption/client-side-encryption.md

```diff
@@ -0,0 +1,2475 @@
+If a collection is present on both the `encryptedFieldsMap` and `schemaMap`, [libmongocrypt](#libmongocrypt) will error
+on initialization. See `fle2-and-fle1-error`.
+
+If a collection has a set of encrypted fields, the behavior of `CreateCollection()` and `Collection.Drop()` is altered.
+An additional helper, `CreateEncryptedCollection()` has been defined as a convenience wrapper around
+`CreateCollection()`. See `fle2-createcollection-drop`.
+
+Automatic encryption in Queryable Encryption is configured with the `encryptedFields`.
+
+If a collection is not present on the `encryptedFields` a server-side collection `encryptedFields` may be used by
+[libmongocrypt](#libmongocrypt). Drivers MUST include the following in the documentation for MongoClient:
+
+> Supplying an `encryptedFieldsMap` provides more security than relying on an `encryptedFields` obtained from the
+> server. It protects against a malicious server advertising a false `encryptedFields`.
+
+#### bypassQueryAnalysis
+
+See [Why is bypassQueryAnalysis needed?](#why-is-bypassqueryanalysis-needed).
+
+### Queryable Encryption Create and Drop Collection Helpers
+
+A collection supporting Queryable Encryption requires an index and three additional collections.
+
+<div id="GetEncryptedFields">
+
+#### Collection `encryptedFields` Lookup (GetEncryptedFields)
+
+The convenience methods support the following lookup process for finding the `encryptedFields` associated with a
+collection.
+
+Assume an exposition-only function $GetEncryptedFields(opts, collName, dbName, askDb)$, where $opts$ is a set of
+options, $collName$ is the name of the collection, $dbName$ is the name of the database associated with that collection,
+and $askDb$ is a boolean value. The resulting `encryptedFields` $EF$ is found by:
+
+1. Let $QualName$ be the string formed by joining $dbName$ and $collName$ with an ASCII dot `"."`.
+2. If $opts$ contains an `"encryptedFields"` property, then $EF$ is the value of that property.
+3. Otherwise, if `AutoEncryptionOptions.encryptedFieldsMap` contains an element named by $QualName$, then $EF$ is the
+   value of that element.
+4. Otherwise, if $askDb$ is `true`:
+   1. Issue a `listCollections` command against the database named by $dbName$, filtered by `{name: <collName>}`. Let
+      the result be the document $L$.
+   2. If $L$ contains an `options` document element, and that element contains an `encryptedFields` document element,
+      $EF$ is $L$`["options"]["encryptedFields"]`.
+   3. Otherwise, $EF$ is *not-found*
+5. Otherwise, $EF$ is considered *not-found*.
+
+#### Create Collection Helper
+
+Drivers MUST support a BSON document option named `encryptedFields` for any
+[create](https://www.mongodb.com/docs/manual/reference/command/create) command helpers (e.g.
+`Database.createCollection()`). This option will be interpreted by the helper method and MUST be passed to the
+[create](https://www.mongodb.com/docs/manual/reference/command/create) command.
+
+> \[!NOTE\]
```

Comment:
Unnecessary escaping. I'll stop commenting on other occurrences, since you can just search for `\[` and fine everything yourself.